build_daemon/lib/daemon.dart

```diff
@@ -101,8 +101,10 @@ class Daemon {
     if (!_doneCompleter.isCompleted) _doneCompleter.complete(exitCode);
   }
 
-  void _createPortFile(int port) =>
-      File(portFilePath(_workingDirectory)).writeAsStringSync('$port');
+  void _createPortFile(int port) {
+    var path = portFilePath(_workingDirectory);
+    File(path).writeAsStringSync('$port');
```

Comment:
Seems like an unnecessary change?

Reply:
ah yeah I had extracted it to a variable for debugging, but don't need it any more